server.py

```diff
@@ -18,13 +21,18 @@ async def create_url(request: Request):
             insert(DATABASE, urljson['url'], urljson['alias'])
```

Comment:
add another check to make sure the alias is only alphanumeric characters, you can use the isalnum() method. if not, raise a ValueError and catch it, throwing HTTPException with code 422